src/dgipy/dgidb.py

```diff
@@ -220,6 +220,75 @@ def get_drug_applications(
     return result
 
 
+def get_clinical_trials(
+    terms: str | list,
+) -> pd.DataFrame:  # TODO: Better error handling for new_row?, use_pandas=False
+    """Perform a look up for clinical trials data for drug or drugs of interest
+
+    :param terms: drug or drugs of interest
+    :return: all clinical trials data for drugs of interest in a DataFrame
+    """
+    base_url = "https://clinicaltrials.gov/api/v2/studies?format=json"
+    rows_list = []
+
+    if isinstance(terms, str):
+        terms = [terms]
+
+    for drug in terms:
+        intr_url = f"&query.intr={drug}"
+        full_uri = base_url + intr_url  # TODO: + cond_url + term_url
```

Comment:
what's the story here?

Reply:
Also, I wonder if we can reduce the size of the response with the `fields` param in case there's anything that we aren't using

Reply:
https://clinicaltrials.gov/data-api/api#:~:text=Studies-,Studies,-Returns%20data%20of

ClinicalTrials.gov api allows you to submit a bunch of different parameter types through the Studies API. For this implementation, I just had the Studies API as the base URL then wanted to leave room for constructing more specific queries if needed just by adding strings together basically. So intr_url corresponds to the query.intervention parameter, but I could also see a world where someone is also interested in supplying the query.condition parameter, or maybe just additional terms as a parameter.

Reply:
For now, just have the intr_url working. 

All this being said, if there is a better way to do this I am all open for it